src/Classes/Utils/ShapeUtils.ts

```diff
@@ -110,11 +111,15 @@ export class ShapeUtils {
     }
 
     private static drawLineShape(context: CanvasRenderingContext2D, length: number, stroke: IStroke): void {
-        context.moveTo(0, -length / 2);
-        context.lineTo(0, length / 2);
-        context.strokeStyle = stroke.color;
-        context.lineWidth = stroke.width;
-        context.stroke();
+        const color = typeof stroke.color === "string" ? ColorUtils.stringToRgb(stroke.color) : ColorUtils.colorToRgb(stroke.color);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Exceeds maximum line length of 120](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5216054)